CorpusSearch/ClientApp/src/routes/NewDocList.tsx

```diff
@@ -0,0 +1,40 @@
+/** NewDocList
+ * Provides a list of links to the latest and greatest files in the Corpus
+ * Which means those document.csv files that git detects as having recent changes
+ */
+import React, {useEffect, useState} from "react"
+function CorpusLink({ident="",name=""}): JSX.Element {
+    if ( name.slice(0,2) != "🎥"){
+        name = "📖" + name
+    }
+    return <div>
+        <a href={"docs/" + ident}>{name}</a>
+    </div>
+}
```

Comment:
This looks better visually if the emoji are not underlined
```patch
Index: CorpusSearch/ClientApp/src/routes/NewDocList.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CorpusSearch/ClientApp/src/routes/NewDocList.tsx b/CorpusSearch/ClientApp/src/routes/NewDocList.tsx
--- a/CorpusSearch/ClientApp/src/routes/NewDocList.tsx	(revision 34a9dabe8c8650db9d3a8f334c6bf50af88d6ea3)
+++ b/CorpusSearch/ClientApp/src/routes/NewDocList.tsx	(date 1715022009532)
@@ -3,14 +3,25 @@
  * Which means those document.csv files that git detects as having recent changes
  */
 import React, {useEffect, useState} from "react"
-function CorpusLink({ident="",name=""}): JSX.Element {
-    if ( name.slice(0,2) != "🎥"){
-        name = "📖" + name
+
+/**
+ * Produces a link to a document, with an emoji denoting the type of content
+ * @example "🎥 Jack as Ned" as a link; no link underline for "🎥 "
+ */
+function CorpusLink({ident = "", name = ""}): JSX.Element {
+    let emojiPrefix = "📖"
+    if (name.slice(0, 2) == "🎥") {
+        name = name.substring(2).trim() // trim the emoji from the name (and any leading spaces)
+        emojiPrefix = "🎥"
     }
+    const href = "docs/" + ident
     return <div>
-        <a href={"docs/" + ident}>{name}</a>
+        {/* Better visuals if the prefix is not underlined; maintain the link for UX */}
+        <a style={{textDecoration: "none"}} href={href}>{emojiPrefix}&nbsp;</a>
+        <a href={href}>{name}</a>
     </div>
 }
+
 export const NewDocList = () => {
 type DocType =  {name:string, ident:string, uploaded:string}
 const [newDocs, setNewDocs  ] 
```

<img width="797" alt="Screenshot 2024-05-06 at 19 59 42" src="https://github.com/david-allison/manx-corpus-search/assets/62114487/8af5416a-4204-43f5-8357-e155fb669740">